src/Util.php

```diff
@@ -29,7 +29,7 @@ private static function getImageResourceFromDB(Image $image): ?ImageResource
     );
   }
 
-  private static function createImageResourceFromCoyoteResource(Image $image, ResourceModel $resource): ImageResource
+  private static function createImageResourceFromCoyoteResource(Image $image, WebhookUpdateModel $resource): ImageResource
```

Comment:
Why was this signature modified? Does this function ever get called with a `WebhookUpdateModel`?

Reply:
I did a search and replace and it all seemed to work. If you think ResourceModel here makes more sense, I will gladly follow that.

Reply:
Global search-and-replace does not sound like a valid strategy without knowing what the code does. `createImageResourceFromCoyoteResource` is used (eventually) [by a call to `Util::getImageResource`](https://github.com/coyote-team/coyote-drupal-8/blob/main/src/Hook/NodeViewAlterPostRenderHook.php#L25), which is not related to Webhook updates. `ResourceModel` is otherwise a valid data model returned by the `CoyoteApiClient` and hence used elsewhere in the module 👍

Reply:
Thanks! Well, you learn by doing is my experience. Util.php reversed